### PR TITLE
Fix --since-time log option

### PR DESCRIPTION
**Pull request by liggitt**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1286116

### CODE REVIEW

**Comment by liggitt** [comment]
cc @kargakis @Miciah @deads2k

File: Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/unversioned/time.go [hunk 1]
```diff
@@ -107,6 +128,16 @@ func (t Time) MarshalJSON() ([]byte, error) {
 	return json.Marshal(t.UTC().Format(time.RFC3339))
 }
 
+// MarshalQueryParameter converts to a URL query parameter value
+func (t Time) MarshalQueryParameter() (string, error) {
```
Comment (kargakis) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/unversioned/time.go]: Don't you need this to be a pointer?

Comment (liggitt) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/unversioned/time.go]: nope :-/
done this way for symmetry with the JSON interface

**Comment by liggitt** [comment]
[test]

**Comment by deads2k** [comment]
got lgtm upstream [merge]

**Comment by liggitt** [comment]
TestClusterResourceOverridePlugin flake, will wait for https://github.com/openshift/origin/pull/7461 before retagging

**Comment by liggitt** [comment]
[merge]

**Pull request merged**